### PR TITLE
Add tests for Project#update_repositories

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/spec/models/project_spec.rb [hunk 1]
```diff
@@ -19,25 +19,39 @@
...
+      it "updates repository attributes of existing repositories" do
+        expect(repository_1.reload.rebuild).to be nil
+      end
+
+      it "imports repository attributes of newly created repositories" do
+        new_repo = project.repositories.find_by(name: "new_repo")
+        expect(new_repo.rebuild).to eq "local"
+        expect(new_repo.block).to eq "never"
+        expect(new_repo.linkedbuild).to eq "all"
+      end
```
Comment (adrianschroeter) [inline src/api/spec/models/project_spec.rb]: I would also verify that "repo_1" has the attributes not set.

Comment (bgeuken) [inline src/api/spec/models/project_spec.rb]: Good point I changed it

**Comment by bgeuken** [comment]
@adrianschroeter I've added the error messages as discussed

I am going to fix the remote error handling in a separate PR soon

**Comment by adrianschroeter** [comment]
+1

**Pull request merged**